resources/sdk/clisdkclient/extensions/cmd/profiles/createprofile.go

```diff
@@ -67,23 +73,35 @@ func requestUserInput() config.Configuration {
 		environment = "mypurecloud.com"
 	}
 
-	for true {
+	fmt.Printf("Please provide either an Access Token, Client Credentials (Client ID and Client Secret) or both\n")
+	fmt.Printf("Note: If you provide an Access Token, this will be used over Client Credentials")
```

Comment:
There needs to be a new line after this. I'm getting `Note: If you provide an Access Token, this will be used over Client CredentialsAccess Token:`